impresion.h

```diff
@@ -19,7 +19,7 @@ class Impresion
     void setTipoPago(TipoPago _tipopago);
 private:
     QPrinter printer;
-    const QString nombre = "recibo.pdf";
+    const QString nombre = "recibo.xps";
```

Comment:
```suggestion
    const QString nombre = "recibo.pdf"; 
```
Esto es en un futuro, porque Qt lanza error en `QPrintDialog` por la impresión nativa en Windows.